Create amenity=health_post

govvin opened the pull request:
Create a global preset for Community or Neighbourhood [Health Posts](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dhealth_post).

govvin committed:
Create health_station-PH.json

There are over 42,000 barangays in the Philippines, and each barangay will have at least one of this facility to serve local communities, in addition to satellite/branches to serve other significant settlements within their jurisdiction.

A large number of these facilities are staffed primarily by trained community healthcare workers known as ["Barangay Health Volunteers"](https://wikipedia.org/wiki/Barangay_health_volunteer).

govvin committed:
Update health_station-PH.json

Add amenity=health_post to addTags section.

govvin committed:
Update health_station-PH.json

Add "bhc", a common acronym for "barangay health center".

govvin committed:
Update health_station-PH.json

Improve JSON formatting

govvin committed:
Update health_station-PH.json

Beautify code? Couldn't find the prettify issue.

govvin committed:
Update health_station-PH.json

Move "Health Center", from term, to alias.

tyrasd commented:
In principle, this looks good to me. :+1: 

Only one question / suggestion: How "bad" would it be to use a more generic name for these amenities, something like _Community Health Post_? If that would be acceptable, we could use that name also for similar presets in [other countries](https://taginfo.openstreetmap.org/tags/amenity%3Dhealth_post#map) which have this kind of healthcare facility. And at the same time it would minimize the effort needed by our translators.

govvin commented:
Not hard at all. :) So, is the current name of this proposed entry going to be an alias , or one of its terms?

> In principle, this looks good to me. +1
> 
> Only one question / suggestion: How "bad" would it be to use a more generic name for these amenities, something like _Community Health Post_? If that would be acceptable, we could use that name also for similar presets in [other countries](https://taginfo.openstreetmap.org/tags/amenity%3Dhealth_post#map) which have this kind of healthcare facility. And at the same time it would minimize the effort needed by our translators.

govvin committed:
Merge branch 'openstreetmap:main' into barangay_health_station

govvin committed:
Add amenity=health_post

Add a globalized preset for "Community Health Post".

govvin committed:
Delete health_station-PH.json

Removed regional preset in favor of a globalized preset for amenity=health_post.

govvin commented on data/presets/amenity/health_post.json:
```diff
@@ -0,0 +1,38 @@
+    ],
+    "moreFields": [
+        "{@templates/poi}",
+        "healthcare",
```
Is it feasible to create a smaller sub-set of the healthcare fields and use that here instead? Any examples I can look at?

govvin committed:
Update health_post.json

Fix hanging comma. Add terms section.

govvin committed:
Create health_post.json

Sub-set of healthcare fields for amenity=health_posts.

govvin committed:
Update health_post.json

Add health_post field, a very limited sub-set of healthcare fields.

govvin committed:
Update health_post.json

Change label from "Type" to "Care Provider".

govvin commented on data/presets/amenity/health_post.json:
```diff
@@ -0,0 +1,42 @@
+    ],
+    "moreFields": [
+        "{@templates/poi}",
+        "healthcare/health_post",
```
![image](https://github.com/openstreetmap/id-tagging-schema/assets/5821688/874a8dc9-ff46-4d49-a0b6-06c3ffd3a655)

Kinda works, but not exactly what I wanted. Other healthcare values still showing up (in black), while those listed in health_post show up as (blue).

matkoniecz replied:
I am not sure, but it seems that `"autoSuggestions": false` (see say https://github.com/openstreetmap/id-tagging-schema/pull/1412/files code) may be used to disable such automatic suggestions

see https://github.com/ideditor/schema-builder/blob/17e6bacf1b575e088d590db72da9566fe624934c/README.md#autosuggestions

matkoniecz commented:
> This pull request is still a work in progress

should it be considered as abandoned? or is it getting close to be worth reviewing again?

govvin commented:
> > This pull request is still a work in progress
> 
> should it be considered as abandoned? or is it getting close to be worth reviewing again?

Thanks for following-up. I think a review is very welcome.  

The tag usage is still growing, while the proposed equivalent healthcare=health_post seems to have stagnated.

govvin commented:
No good reason, other than kinda forgetting about it. It's been two year since this PR was drafted.

Thanks @matkoniecz ,  for the comments above, and the possible fix to the issue I mentioned.

I'm a bit busy now, but I'll look into this again in a week or two, and prepare a new PR.

matkoniecz commented:
It waited literally for years so I feel a bit silly pinging about it - but have you looked into it by any chance?

(I am trying to help untuck PRs to avoid problem of PRs sitting stale for years)

govvin committed:
Merge branch 'openstreetmap:main' into barangay_health_station

govvin committed:
Update health_post.json

govvin committed:
Update health_post.json

govvin committed:
Merge branch 'openstreetmap:main' into barangay_health_station

govvin committed:
Update health_post.json

Add option for BNS workers.

govvin committed:
Update health_post.json

Revert to original options

govvin committed:
Update health_post.json

Remove hanging comma.

govvin committed:
Update health_post.json

Re-word label for global users.

govvin committed:
Update health_post.json with additional fields

Added description, required, allow_custom, custom_key, options_order, strings, examples, and version fields to the health_post JSON.

govvin committed:
Rename 'Barangay Health Station' to 'Community Health Post'

add similar terms, to help improve search

govvin committed:
Drop none and unknown from options_order and bump version to 0.3

govvin committed:
Refactor healthcare post JSON structure

Removed custom options and updated version number.
